### ISSUE TITLE
investigate BlazeFace

### ISSUE THREAD

**Issue by shiffman** [shiffman/BodyBlock#4]
Thanks to https://twitter.com/ldenoue/status/1267702800173223939

https://github.com/tensorflow/tfjs-models/blob/master/blazeface/README.md

**Comment by shiffman** [comment]
Thanks to @adamhrv other models to investigate:

https://twitter.com/adamhrv/status/1268462995266506753

> Is there a TF.js version of TinyFaces, RetinaFace, or YoloV4? Less performant, but only 1MB there is also https://github.com/Linzaer/Ultra-Light-Fast-Generic-Face-Detector-1MB that could perhaps be ported to JS?

**Comment by joeyklee** [comment]
@shiffman - I suppose if the point is to focus on masking faces, we could/should use a lib specifically for face detection? The Tiny Face model of face-api is actually pretty snappy.

**Comment by shiffman** [comment]
yes, I was thinking of adding this in as a second pass to catch faces that BodyPix misses. Then we could have an interface with checkboxes for which models you want to run (one or both):

* BodyPix
* TinyFace

And maybe we offer more models that people can try some might work better with different videos.

And also options for:

* Block faces
* Block bodies

So you can choose one or both. . .

**Comment by joeyklee** [comment]
That's great + makes perfect sense.

**Comment by shiffman** [comment]
Great! I am going to work on adding TinyFace in a separate branch, more soon!

**Comment by shiffman** [comment]
Follow in #17
